COMPATIBILITY.md

```diff
@@ -69,4 +69,4 @@ For cases when a first-class Workflow step (rather than an adaptation of functio
 
 - [X] `parallel-test-executor`: supported with `splitTests` step since 1.6
 - [ ] `gerrit-trigger`: [JENKINS-26102](https://issues.jenkins-ci.org/browse/JENKINS-26102), [JENKINS-26103](https://issues.jenkins-ci.org/browse/JENKINS-26103)
-- [ ] `mailer`: [JENKINS-26104](https://issues.jenkins-ci.org/browse/JENKINS-26104)
+- [X] `mailer` (`workflow`): supported with `mail` step since 1.2
```

Comment:
The new `mail` step is in the `workflow` plugin (not the `mailer` plugin).

My interpretation of the format in this doc is that the first token is the name of the step, then the second is the name of the plugin (i.e. `workflow` here). So I should probably change the first token from `mailer` to `mail`.

I do agree though that the current format is not too clear and can be misinterpreted.